opentelemetry/proto/common/v1/common.proto

```diff
@@ -21,6 +21,36 @@ option java_package = "io.opentelemetry.proto.common.v1";
 option java_outer_classname = "CommonProto";
 option go_package = "go.opentelemetry.io/proto/otlp/common/v1";
 
+// inlined parquet-gen options
+import "google/protobuf/descriptor.proto";
+
+message MessageOptions {
```

Comment:
Please add a comment to explain the purpose of this message or other new messages introduced in this PR.

Reply:
Well, there's a comment line 24 -- I guess what you're saying is that it's not cutting it

Reply:
I think it would be great to have more comments to explain what is going on here. I am not familiar with Parquet and probably some other contributors to this repos also aren't. We need some help to understand the change and also be able to maintain it in the future.

Reply:
OK I have added more information.